### PR TITLE
Change script name, add argparser

### PR DESCRIPTION
**Pull request by mattsmart**

### CODE REVIEW

File: models/tridimensional/results_csv.py [hunk 1]
```diff
@@ -10,9 +11,13 @@ def get_scores(filedata):
     return [filedata[i].split(" ")[-1][:-1] for i in xrange(4)]
```
Comment (jrhawley) [inline models/tridimensional/results_csv.py]: Want to add a comment block saying what the arguments are and an example?

Comment (mattsmart) [inline models/tridimensional/results_csv.py]: That was from the last PR! lol i'll do it though

**Comment by mattsmart** [comment]
@jrhawley Commit is up

**Comment by jrhawley** [comment]
Cool, works as expected. Merging

**Pull request merged**